pages/1.13/monitoring/logging/aggregating/elk/index.md

```diff
@@ -10,144 +10,82 @@ enterprise: false
 
 
 
-You can pipe system and application logs from the nodes in a DC/OS cluster to an Elasticsearch server. These instructions are based on CentOS 7 and might differ substantially from other Linux distributions.
+You can pipe system and application logs from the nodes in a DC/OS cluster to an Elasticsearch server.
 
 
 # What this document does and does not cover
 
-This document describes how to send Filebeat output from each node to a centralized Elasticsearch instance. This document describes how to directly stream from Filebeat into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][8] and the example in [Filtering logs with ELK][3].
+This document describes how to send Fluent Bit output from each node to a centralized Elasticsearch instance. This document describes how to directly stream from Fluent Bit into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][4] and the example in [Filtering logs with ELK][2].
```

Comment:
Rewrite `This document describes how to send Fluent Bit output from each node to a centralized Elasticsearch instance. This document describes how to directly stream from Fluent Bit into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][4] and the example in [Filtering logs with ELK][2].`

with `This page describes how to send Fluent Bit output from each node to a centralized Elasticsearch instance. It also describes how to directly stream from Fluent Bit into Elasticsearch. Logstash is not used in this architecture. If you are interested in filtering, parsing, and understanding the logs with an intermediate Logstash stage, see the Logstash [documentation][4] and the example in [Filtering logs with ELK][2].